source/MaterialXView/ShaderValidators/Osl/OslValidator.cpp

```diff
@@ -175,9 +266,16 @@ void OslValidator::validateCreation(const std::vector<std::string>& stages)
     {
         fileName += ".osl";
     }
+
+    // TODO: Seems testrender will crash currently when trying to convert to "object" space.
```

Comment:
testrender will crash with transform to object space for some reason. Making all "object" space assignments to "world" space. I know this is not the most reliable way but can't really parse the code easily and spaces are hard-coded to be object in a number of places and can't be easily changed by setting parameters. e.g. triplanar mapping is hard-coded to take object space positions for some reason and there is no parameter that is accessible to modify this in the test files.